src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py

```diff
@@ -2030,6 +2030,7 @@ def _get_network_plugin(self, enable_validation: bool = False) -> Union[str, Non
         """
         # read the original value passed by the command
         network_plugin = self.raw_param.get("network_plugin")
+        network_plugin_mode = self.raw_param.get("network_plugin_mode")
```

Comment:
Since this is a preview feature, no such option/parameter in azure-cli/acs. Recommend to dup the function in aks-preview.

Reply:
Opened this PR [#5780](https://github.com/Azure/azure-cli-extensions/pull/5780) to move the changes to aks-preview.